test/yargs-parser.js

```diff
@@ -762,6 +762,29 @@ describe('yargs-parser', function () {
       argv.should.have.property('n', 123)
     })
 
+    it('should set n to the numeric value 123, with n at the end of a group', function () {
+      var argv = parser([ '-ab5n123' ])
```

Comment:
To me, option `.2.` feels a little more useful, I can't think of a situation where I want `5: true` but I can imagine a lazy person writing `-x1y5` and expecting:

`{x: 1, y:5}`